scalastyle-config.xml

```diff
@@ -82,11 +82,6 @@
    <parameter name="doubleLineAllowed"><![CDATA[false]]></parameter>
   </parameters>
  </check>
- <check level="error" class="org.scalastyle.scalariform.MethodLengthChecker" enabled="true">
-  <parameters>
-   <parameter name="maxLength"><![CDATA[50]]></parameter>
```

Comment:
this check can be annoying but it is usually pointing to something that needs looking at

Reply:
In the case it was complaining about I thought it was a reasonable function